assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -224,28 +252,45 @@ export default function UserInputSelectOptions( {
 						{ __( 'Other:', 'google-site-kit' ) }
 					</ListComponent>
 
-					<TextField
-						label={ __(
-							'Type your own answer',
-							'google-site-kit'
-						) }
-						noLabel
-					>
-						<Input
-							id={ `${ slug }-select-options` }
-							value={ other }
-							onChange={ onOtherChange }
-							ref={ inputRef }
-							disabled={ disabled }
-							tabIndex={
-								! values.includes( other.trim() ) || ! isActive
-									? '-1'
-									: undefined
+					<div className="googlesitekit-user-input__select-option-text-field">
```

Comment:
Nice catch, thank you! I've updated this text field to have a fluid-width so that it adapts to the container width, but with a maximum of `250px` (current fixed width).

Reply:
Thanks, that's much better. There is still one slight visual hiccup, which is the placeholder text is truncated at narrow viewports. Here it is at 320px (in the plugin as opposed to Storybook which actually provides less than 320px due to the margins added to the story):

![image](https://user-images.githubusercontent.com/18395600/203538520-17c16ce4-11d5-4d74-99ae-d3f16a91d317.png)

I'd suggest adding the following CSS to provide ellipses in this scenario:

```css
input:placeholder-shown {
	text-overflow: ellipsis;
}
```

Resulting in:

![image](https://user-images.githubusercontent.com/18395600/203539323-220fb1d4-f6b6-4cd7-96c9-fd487f4f1b44.png)

If necessary we can always iterate further on a separate issue...